changelog/issue-3615.md

```diff
@@ -0,0 +1,64 @@
+audience: general
+level: major
+reference: issue 3615
+---
+RFC 165 has been implemented, allowing for greater administrator control over
+"public" endpoints. Previously these were guarded by no scopes and could be
+accessed by anyone with no way to limit this. In this release all
+unauthenticated API calls are now granted the scope `assume:anonymous`.
+Additionally, most previously unprotected endpoints are now guarded by at
+least one scope, to enable the following:
+
+* To maintain current behavior, grant the `anomymous` role the following scopes:
+
+  - `auth:current-scopes`
+  - `auth:expand-scopes`
+  - `auth:get-client:*`
+  - `auth:get-role:*`
+  - `auth:list-clients`
+  - `auth:list-roles`
+  - `github:consume-webhook`
+  - `github:get-badge:*`
+  - `github:get-repository:*`
+  - `github:latest-status:*`
+  - `github:list-builds`
+  - `hooks:get:*`
+  - `hooks:list-hook-groups`
+  - `hooks:list-hooks:*`
+  - `hooks:list-last-fires:*`
+  - `hooks:status:*`
+  - `hooks:trigger-hook:*`
+  - `index:find-task:*`
```

Comment:
what about `index:list-namespaces*`?  Are there other scopes omitted here?

Reply:
Good catch. It looks like the scopes that had `if ... else` were not caught by the grep command.